Index: b/kstars/kstars/indi/libs/apogee/ApogeeUsbLinux.cpp
===================================================================
--- a/kstars/kstars/indi/libs/apogee/ApogeeUsbLinux.cpp
+++ b/kstars/kstars/indi/libs/apogee/ApogeeUsbLinux.cpp
@@ -3,7 +3,6 @@
 
 #include <assert.h>
 #ifndef OSX
-#include <sys/io.h>
 #endif
 #include <sys/time.h>                                                           
 #include <sys/resource.h>
