Index: b/kstars/kstars/indi/CMakeLists.txt
===================================================================
--- a/kstars/kstars/indi/CMakeLists.txt
+++ b/kstars/kstars/indi/CMakeLists.txt
@@ -1,5 +1,7 @@
 KDE4_NO_ENABLE_FINAL(indi)
 
+find_package(ZLIB REQUIRED)
+
 include_directories( ${CMAKE_CURRENT_BINARY_DIR}/../.. )
 include_directories( ${CMAKE_SOURCE_DIR}/kstars/kstars/indi/libs)
 include_directories( ${CMAKE_SOURCE_DIR}/kstars/kstars/indi/libs/apogee)
@@ -409,7 +411,7 @@
 
 kde4_add_executable(indi_fli_ccd ${fliccd_SRCS} ${libfli_SRCS} ${liblilxml_SRCS} ${libindicom_SRCS})
 
-target_link_libraries(indi_fli_ccd ${KDE4_KDECORE_LIBS} ${CFITSIO_LIBRARIES} m)
+target_link_libraries(indi_fli_ccd ${KDE4_KDECORE_LIBS} ${CFITSIO_LIBRARIES} m ${ZLIB_LIBRARY})
 
 if (NOVA_FOUND)
   target_link_libraries(indi_fli_ccd ${NOVA_LIBRARIES})
@@ -431,7 +433,7 @@
 
 kde4_add_executable(indi_apogee_alta ${apogee_alta_SRCS} ${libfli_SRCS} ${liblilxml_SRCS} ${libindicom_SRCS})
 
-target_link_libraries(indi_apogee_alta ${KDE4_KDECORE_LIBS} ${LIBUSB_LIBRARIES} ${CFITSIO_LIBRARIES})
+target_link_libraries(indi_apogee_alta ${KDE4_KDECORE_LIBS} ${LIBUSB_LIBRARIES} ${CFITSIO_LIBRARIES} ${ZLIB_LIBRARY} )
 
 if (NOVA_FOUND)
   target_link_libraries(indi_apogee_alta ${NOVA_LIBRARIES})
@@ -453,7 +455,7 @@
 
 kde4_add_executable(indi_sbig_ccd ${sbigccd_SRCS} ${liblilxml_SRCS} ${libindicom_SRCS})
 
-target_link_libraries(indi_sbig_ccd ${KDE4_KDECORE_LIBS} ${CFITSIO_LIBRARIES} ${SBIG_LIBRARIES})
+target_link_libraries(indi_sbig_ccd ${KDE4_KDECORE_LIBS} ${CFITSIO_LIBRARIES} ${SBIG_LIBRARIES} ${ZLIB_LIBRARY})
 
 if (NOVA_FOUND)
   target_link_libraries(indi_sbig_ccd ${NOVA_LIBRARIES})
@@ -480,7 +482,7 @@
 
 kde4_add_executable(indi_sbig_stv NOGUI ${stv_SRCS} ${liblilxml_SRCS} ${libindicom_SRCS})
 
-target_link_libraries(indi_sbig_stv ${KDE4_KDECORE_LIBS} ${CFITSIO_LIBRARIES} ${NOVA_LIBRARIES})
+target_link_libraries(indi_sbig_stv ${KDE4_KDECORE_LIBS} ${CFITSIO_LIBRARIES} ${NOVA_LIBRARIES} ${ZLIB_LIBRARY})
 
 install(TARGETS indi_sbig_stv ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
@@ -502,7 +504,7 @@
 
 kde4_add_executable(indi_meade_lpi ${meade_lpi_SRCS} ${libwebcam_SRCS} ${liblilxml_SRCS} ${libindicom_SRCS})
 
-target_link_libraries(indi_meade_lpi ${KDE4_KDECORE_LIBS}  ${CFITSIO_LIBRARIES})
+target_link_libraries(indi_meade_lpi ${KDE4_KDECORE_LIBS}  ${CFITSIO_LIBRARIES} ${ZLIB_LIBRARY})
 
 if (NOVA_FOUND)
   target_link_libraries(indi_meade_lpi ${NOVA_LIBRARIES})
